modules/managing-dedicated-administrators.adoc

```diff
@@ -16,21 +16,16 @@ Administrator roles are managed using a `cluster-admin` or `dedicated-admin` gro
 .Procedure
 
 . Navigate to the *Cluster Details* page and *Access Control* tab.
-. Click the *Add user* button (first user only).
+. Select the *Cluster Roles and Access* tab and click the *Add user* button.
 . Enter the user name and select the group.
-. Click the *Add* button.
+. Click the *Add user* button.
```

Comment:
I know it was already like that, but same comment about not saying "button" in case you care to adjust here too :)